docs/typescript/how-to-get-the-result-of-a-workflow-execution-in-typescript.md

```diff
@@ -0,0 +1,55 @@
+---
+id: how-to-get-the-result-of-a-workflow-execution-in-typescript
+title: How to get the result of a Workflow execution in Typescript
+sidebar_label: Get the result of a Workflow execution
+description: Get the result of a Workflow execution
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+To return the results of a Workflow Execution:
+
+```typescript
+return (
+  'Completed ' +
+  wf.workflowInfo().workflowId +
+  ', Total Charged: ' +
+  totalCharged
+);
+```
+
+`totalCharged` is just a function declared in your code. For a full example, see [subscription-workflow-project-template-typescript/src/workflows.ts](https://github.com/temporalio/subscription-workflow-project-template-typescript/blob/main/src/workflows.ts).
+
+Workflow functions may or may not return a result when they complete.
```

Comment:
Wait a sec - above I read that only Query handlers can return a result in TypeScript?